appdata/subsurface.appdata.xml

```diff
@@ -1,34 +1,47 @@
    weights and exposure protection used, divemasters and dive buddies, etc.
-   Subsurface also enables the users to rate their dives and provide additional dive notes.
-  </p>
-  <p>
-   Dives can be downloaded from a variety of dive computers, inserted manually or imported from other programs.
+   Subsurface also enables the users to rate their dives and provide additional dive notes.</p>
+    <p>Dives can be downloaded from a variety of dive computers, inserted manually or imported from other programs.
    A wide array of diving statistics and information is calculated and displayed, like the user’s SAC rate,
-   partial pressures of O2, N2 and He, calculated deco information, and many more.
-  </p>
-  <p>
-   The dive profiles (and the tank pressure curves) can be visualized in a comprehensive and clean way, that
+   partial pressures of O2, N2 and He, calculated deco information, and many more.</p>
+    <p>The dive profiles (and the tank pressure curves) can be visualized in a comprehensive and clean way, that
    provides the user with additional information on relative velocity (and momentary air consumption) during the dive.
    Subsurface also allows the user to print out a detailed log book that includes dive profiles and other relevant information.
-   The program is localized in about 20 languages and well supported by an active developer community.
-  </p>
- </description>
- <screenshots>
-  <screenshot type="default" width="1860" height="1046">http://subsurface-divelog.org/screenshots/main.png</screenshot>
-  <screenshot width="778" height="438">http://subsurface-divelog.org/screenshots/preferences.png</screenshot>
-  <screenshot width="782" height="440">http://subsurface-divelog.org/screenshots/preferences2.png</screenshot>
- </screenshots>
- <url type="homepage">http://www.http://subsurface-divelog.org/</url>
- <updatecontact>subsurface@subsurface-divelog.org</updatecontact>
-</application>
+   The program is localized in about 20 languages and well supported by an active developer community.</p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://subsurface-divelog.org/screenshots/main.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://subsurface-divelog.org/screenshots/preferences.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://subsurface-divelog.org/screenshots/preferences2.png</image>
+    </screenshot>
+  </screenshots>
+  <url type="homepage">https://subsurface-divelog.org</url>
+  <url type="bugtracker">https://github.com/Subsurface-divelog/subsurface/issues</url>
+  <url type="faq">https://subsurface-divelog.org/faq/</url>
+  <url type="help">https://subsurface-divelog.org/documentation/</url>
+  <url type="translate">https://www.transifex.com/subsurface/subsurface/</url>
+  <releases>
+     <release version="4.8.1" date="2013-07-14" />
```

Comment:
Hardcoding the release here seems unfortunate... this should be implemented differently.

Reply:
Would removing these values and adding this to scripts/ be acceptable?

```
#!/bin/sh
version=$(git describe --abbrev=12)
date=$(git log -1 --format="%at" | xargs -I{} date -d @{} +%Y-%m-%d)
sed -e "s|<release version=\"\" date=\"\" />|<release version=\"$version\" date=\"$date\" />|" -i appdata/subsurface.appdata.xml
```

Reply:
yes, we have a couple of places where we do similar things for example for other builds

Reply:
Okay, done